KubeArmor/feeder/policyMatcher.go

```diff
@@ -960,7 +973,8 @@ func (fd *Feeder) UpdateMatchedPolicy(log tp.Log) tp.Log {
 		}
 
```

Comment:
Can we document what's happening in this particular switch case (`case "Network":`) so as future contributors have context atleast for this part?

Reply:
sure, will add

Reply:
some comments were already added, at specific conditions